sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -111,11 +112,15 @@ def _prepare_all_metrics(self, metrics):
       insert_rows.append(counter_dict)
 
     dists = metrics['distributions']
-    if len(dists) > 0:
-      runtime = RuntimeMetric(dists, submit_timestamp, metric_id)\
-        .as_dict()
-      insert_rows.append(runtime)
-
+    sorted_dists = sorted(dists, key=lambda x: x.key.metric.name)
```

Comment:
For a performance reason, it'd be worth to use `attrgetter` from `operator` module as a key parameter. 
The same applies to the instruction below.

Reply:
If size of `dists` is not that large, we can as well keep current version as it may be more readable.

Reply:
This is good advice Kamil, thank you. But I must agree with Valentyn. Because of rather small arrays we won't get much out of performance by using `attrgetter` and as I checked how it would look like it may be a little bit less readable.